scrapers/nsw/canterbury_bankstown.py

```diff
@@ -0,0 +1,53 @@
+import sys
+from pathlib import Path
+
+parent_dir = str(Path(__file__).resolve().parent.parent.parent)
+if parent_dir not in sys.path:
+    sys.path.append(parent_dir)
+
+from base_scraper import BaseScraper, register_scraper
+from logging.config import dictConfig
+from _dataclasses import ScraperReturn
+from bs4 import BeautifulSoup
+import re
+
+@register_scraper
+class CanterburyBankstownScraper(BaseScraper):
+    def __init__(self):
+        council = "canterbury_bankstown"
+        state = "NSW"
+        base_url = "https://www.cbcity.nsw.gov.au/"
+        super().__init__(council, state, base_url)
+
+    def scraper(self) -> ScraperReturn:
+        self.logger.info(f"Starting {self.council_name} scraper")
+
+        name = None
+        date = None
+        time = None
```

Comment:
Seems the time is fixed to 6:30pm for Canterbury-Bankstown, would be best to code it in then for completeness sake. If it changes in the future this can be revised then.
```suggestion
        time = "18:30"
```